queries/boundaries.jinja2

```diff
@@ -101,16 +100,14 @@ UNION ALL
 
 SELECT
   gid AS __id__,
-  'openstreetmap.org' AS source,
-  NULL AS name,
-  8 as min_zoom,
-  true AS maritime_boundary,
   {% filter geometry %}{{ bounds['polygon']|bbox_intersection('the_geom',3857) }}{% endfilter %} AS __geometry__,
-  '{"kind": "maritime"}'::json as mz_properties,
-  hstore(ARRAY[
-           'source', 'openstreetmap.org',
-           'kind', 'maritime']) AS mz_additional_tags,
-  NULL::json AS tags
+  json_build_object('kind', 'maritime') as mz_properties,
+  jsonb_build_object(
+    'source', 'openstreetmap.org',
+    'min_zoom', 8,
+    'maritime_boundary', true,
+    'kind', 'maritime'
```

Comment:
Does `kind` need to be in both `__properties__` and `mz_properties`? Could we move `maritime_boundary` into `mz_properties`?